argilla/docs/tutorials/text_classification.ipynb

```diff
@@ -0,0 +1,529 @@
+{
```

Comment:
<p>We can train the model using our previous function, but this time with a high-quality human-annotated training set.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>